Versioning.yml

```diff
@@ -1,2 +1,3 @@
 v1.1.1:
-  useeior: "SEF_v1.1"
+  useeior_tag: "SEF_v1.1"
+  useeior_ver: "1.0.0"
```

Comment:
> I see - I guess this is because of our error where we didn't update the DESCRIPTION version number with that release https://github.com/USEPA/useeior/blob/8eb2234a9ca6924655c0ce369bdb76a654869bc6/DESCRIPTION#L4

That's right unfortunately.